pkg/skaffold/deploy/helm.go

```diff
@@ -62,9 +63,7 @@ type HelmDeployer struct {
 	*latest.HelmDeploy
 
 	kubeContext string
-	kubeConfig  string
-	namespace   string
-	forceDeploy bool
+	opts        config.SkaffoldOptions
```

Comment:
we try where we can to avoid passing around resources that aren't necessary. is there a certain field you need from the `SkaffoldOptions` that you could grab out and embed in the `HelmDeployer` rather than just putting the whole opts in there?

Reply:
Rolled back those changes